admin.md

```diff
@@ -1,112 +1,145 @@
 
- Navigate to the ElasticSearch service instance in your [IBM Cloud account](https://cloud.ibm.com/login). In the example below, the ElasticSearch Service is named `HRI-DocumentStore`.
+1. Navigate to the Elasticsearch service instance in your [IBM Cloud account](https://cloud.ibm.com/login). In the example below, **HRI-DocumentStore** is the Elasticsearch service. 
 
- Click on the "Service Credentials" link on the left-hand side ElasticSearch service menu. 
-  
+2. In Elasticsearch, in the left menu, click **Service Credentials**.
+
+**Figure 1: Elasticsearch menu**
    ![doc-store-svc-creds](assets/img/doc_store_svc_creds.jpg)
-  
- On the Service Credentials page click the New Credential button:
-  
-   ![doc-store-new-cred](assets/img/doc_store_new_cred.jpg)
-    
- You will name your new credential and click the "Add" button. After that, your credential will be generated for you and you will be able to click on the "View Credentials" link (circled in Red below):
- 
+
+3. On the Service Credentials page, click **New Credential**.
+
+4. Type a name for the credentials.
+
+5. Click **Add**. This generates your credentials. To view them, click **View Credentials** (Figure 2).
+   
+   **Figure 2: Viewing credentials**
    ![view-new-credential](assets/img/view_new_credential.jpg)
+
+6. From the expanded service credentials window, retrieve your newly-created Elasticsearch username and password. You need these for the Elasticsearch REST commands using Curl. 
    
-   + From the expanded service credentials window, retrieve your newly created ElasticSearch Username and Password needed for the ElasticSearch REST commands using cURL. Capture the string that contains both the username and password by copying the `-u` "argument" from the JSON string (within the "connection" and "cli" sections). 
+   Capture the string that contains both the username and password by copying the ```-u``` "argument" from the JSON string, in the ```connection``` and ```cli``` sections). 
    
-   For example, in the screen cap above, the new credential is named `test-cred-1`, and, in the circled section after the word "arguments\" :[", there is a long string (after the line with "-u\") starting with the characters `ibm_cloud_40cfdf40`. The section of this string before the ":" (colon) is the UserName you will need; while the section after the ":" is the Password, as in:
-    
-       UserName: ibm_cloud_40cfdf40_e075_44aa_8bc1_04a247619e4f
-       Password: 79d9379ee59771fdf544c84538879f5ef6446e73c0707a768d031cd86e1e2020
+   **Example**
+   
+   In Figure 2, in the **Key Name** column, the new credential is `test-cred-1`. In the circled section after **arguments** :[", after the line with **-u**, you'll see a string starting with `ibm_cloud_40cfdf40`. The section of this string before the colon (:) is the UserName that you need. The section after the colon is the password. 
+   
+   **Figure 3: Example of the UserName and Password**
```

Comment:
For some reason the formatting of the "Figure 3" code below is not showing up now as a code block. It should look more like:

`UserName: ibm_cloud_40cfdf40_e075_44aa_8bc1_04a247619e4f Password: 79d9379ee59771fdf544c84538879f5ef6446e73c0707a768d031cd86e1e2020
`

Reply:
![fix](https://user-images.githubusercontent.com/56731875/99990950-4bb68180-2d69-11eb-8653-f35152b708e9.jpg)
I've removed a line break, moving "Password" up to the previous line.